src/libs/settings/creportersavedstate.h

```diff
@@ -34,6 +34,7 @@ class CReporterSavedState: public CReporterSettingsBase
     Q_OBJECT
 
     Q_PROPERTY(quint32 crashNotificationId READ crashNotificationId WRITE setCrashNotificationId NOTIFY crashNotificationIdChanged)
+    Q_PROPERTY(quint32 storageUsageNotificationId READ storageUsageNotificationId WRITE setStorageUsageNotificationId NOTIFY storageUsageNotificationIdChanged)
```

Comment:
A bit pondering why these are Q_PROPERTIES when usage is only couple references in cpp, but oh well.

Reply:
Maybe it was supposed to be used from QML

Reply:
My guess is more of cargo cult programming.

Reply:
Possibly. Let's keep this question out of the scope.